.github/workflows/publish_jobs.yml

```
@@ -0,0 +1,34 @@
+# This workflow publishs all Jenkins jobs when a PR is merged.
+#
+---
+on:
+  push:
+    branches:
+      - main
+    paths:
+      # Run only when the definitions of jobs changed.
+      - 'jobs-builder/jobs/**'
+name: Publish Jenkins jobs
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install the Jenkins Job Builder
+        run: |
+          pip install --user jenkins-job-builder
+      - name: Run the publish script
+        env:
+          JENKINS_USER: ${{ secrets.JENKINS_USER }}
+          JENKINS_PASSWORD: ${{ secrets.JENKINS_PASSWORD }}
+        working-directory: jobs-builder
+        run: |
+          cp jjb.conf.template jjb.conf
+          if [[ -z "$JENKINS_USER" || -z "$JENKINS_PASSWORD" ]]; then
+              echo "Missing the secrets JENKINS_USER and/or JENKINS_PASSWORD"
+              exit 1
+          fi
+          sed -i -e "s/user=XXX/user=$JENKINS_USER/" jjb.conf
+          sed -i -e "s/password=XXX/password=$JENKINS_PASSWORD/" jjb.conf
```

Nit: You could chain these into a single call:

```suggestion
          sed -i \
              -e "s/user=XXX/user=$JENKINS_USER/" \
              -e "s/password=XXX/password=$JENKINS_PASSWORD/" \
              jjb.conf
```

---

Done. Thanks! Old habits are hard to die! :) sometimes I find myself doing `cat file | grep word` still...